### DISCUSSION TITLE
I want to improve SyncPlay, where do I start ?

### DISCUSSION THREAD

**Discussion by LelouBil** [jellyfin/jellyfin#10678] [Q&A] [answered]
Hello, 
It's been 5 months since I installed Jellyfin and it's a wonderful software.
However, I am running into a lot of SyncPlay issues that I am not sure are reported here, mainly with the web ui. (like syncplay freezing completely and spinning infinitely)
 I have experience working in C# and web development and I want to improve on it, so I have a couple of questions.

Would this kind of issue be only web, only server or both ? 
What's the status for SyncPlay, are there things already planned ?
What do I need to know to get started ?
Should I also (or instead) ask this in the jellyfin-web repo ?

**Comment by felix920506** [comment, marked as answer]
Hi, if you would like to work on syncplay, you can start by searching "syncplay" on the repo. The results will tell you everything about the code.

We would also appreciate your contribution to the documentation on the feature while you are digging through it, as the original author basically abandoned it without leaving documentation. 

We wish you best of luck!
